chris_backend/uploadedfiles/views.py

```diff
@@ -130,4 +130,6 @@ def get(self, request, *args, **kwargs):
         Overriden to be able to make a GET request to an actual file resource.
         """
         user_file = self.get_object()
-        return FileResponse(user_file.fname)
+        response = FileResponse(user_file.fname)
+        response["Content-Length"] = len(user_file.fname)
```

Comment:
I think it would be more clear if you directly called `user_file.fname.size`

Reply:
Ok @jennydaman .